### ISSUE TITLE
Request - Allow different option for NG+

### ISSUE THREAD

**Issue by khovel** [Kitteh6660/Corruption-of-Champions-Mod#1115]
### Issue type
- [ ] Feature request

Prior to the Ascension feature, there was a basic NG+ where you kept your level and stats, but enemies didn't gain a level boost based on the number of ascensions. 

Can you add into here a variation to bring that back? 

Something on the lines of Reincarnate. 

Reincarnate: 
- Level resets to 1, perks( or perk points ) and stats remain
- Only items that retain are currently equipped garments ( no weapons or armor ) 
- Storage chests that were found remain, but storage is emptied
- Body remains as is at the end of prior playthrough
- Game difficulty increases,( normal -> hard -> nightmare -> extreme ) 

Basically an Ascension where level, inventory, weapons/armor reset, game difficulty goes up a step, but enemy levels do not increase. 

I find this game enjoyable, but once you ascend, it becomes a bit more tedious and grindy. 

Side note, once we build the cabin, can that just be defaulted that the player is in it once built, rather than having to go to it every night. Make it a flag so that all interactions with camp npcs dont have to go through an extra menu to leave the cabin
